### PR TITLE
snakemake bugs

### PR DESCRIPTION
**Pull request by kennyworkman**

### CODE REVIEW

File: latch_cli/centromere/ctx.py [hunk 1]
```diff
@@ -120,6 +120,11 @@ def __init__(
                                 image_name=self.task_image_name(entity.name),
                                 pkg_dir=entity.dockerfile_path.parent,
                             )
+                if hasattr(self, "workflow_name"):
```
Comment (ayushkamat) [inline latch_cli/centromere/ctx.py:123]: `click.secho(..., fg="red")` and then `raise click.Exit(1)` instead of throwing a valueerror

Comment (ayushkamat) [reply latch_cli/centromere/ctx.py:123]: see error messaging on latch cp for examples of what i mean

File: latch_cli/centromere/ctx.py [hunk 2]
```diff
@@ -205,8 +212,14 @@ def __init__(
                                     click.secho("Failed to open file", fg="red")
                         sys.exit(1)
 
-                assert metadata._snakemake_metadata is not None
-                assert metadata._snakemake_metadata.name is not None
+                if (
+                    metadata._snakemake_metadata is None
+                    or metadata._snakemake_metadata.name is None
+                ):
+                    raise ValueError(
```
Comment (ayushkamat) [inline latch_cli/centromere/ctx.py:219]: same

File: latch_cli/services/init/common/.dockerignore [hunk 3]
```diff
@@ -3,7 +3,7 @@ README.md
 CHANGELOG.md
 docker-compose.yml
 Dockerfile
-.latch
+.latch/.logs
```
Comment (ayushkamat) [inline latch_cli/services/init/common/.dockerignore:6]: del, we don't wanna copy over things like ssh keys or whatever else we store in `.latch`

Comment (ayushkamat) [reply latch_cli/services/init/common/.dockerignore:6]: if you want to explicitly only include the jit entrypoint do something like this: https://stackoverflow.com/a/51429852

**Review by ayushkamat — CHANGES_REQUESTED** [review]

File: latch_cli/centromere/ctx.py (hunk 2 above)
Comment (kennyworkman) [reply latch_cli/centromere/ctx.py:221]: i copied max's asserts

**Pull request merged**